### DISCUSSION TITLE
Error indicating trend arrow

### DISCUSSION THREAD

**Discussion by dinizmauricio** [NightscoutFoundation/xDrip#3182] [Bug reports]
I'm using XDrip + Aaps + NightScout. I noticed that the trend arrow is different in Xdrip and the other apps.
When searching for data in Xdrip, via webservice, I was able to see that the correct arrow is not being generated.

Because of this, NightScout and AAps are also generating unreliable arrows.

![screenArrow](https://github.com/NightscoutFoundation/xDrip/assets/17601929/effcda80-aa1a-4901-a9aa-8c911bdfa545)

Could someone check?

Maurício

**Comment by Navid200** [comment]
Have you seen these?
https://github.com/NightscoutFoundation/xDrip/issues/414
https://github.com/NightscoutFoundation/xDrip/issues/800
https://github.com/NightscoutFoundation/xDrip/issues/691

**Comment by dinizmauricio** [comment]
@Navid200, Thanks for the information, it appears to be case #691.

**Comment by dinizmauricio** [comment]
In short: Xdrip shows an arrow on the screen and application and another for those who consult via the webservice. The one on the screen is correct, but the one on the webservice does not match reality.
